src/App.js

```diff
@@ -1,7 +1,25 @@
 import './App.css';
+import {useEffect, useState} from 'react';
+import MenuList from './components/MenuList';
+
 
 function App() {
-  return <div className="App">hello, world!</div>;
-}
+  const [menuData, setMenuData] = useState([]);
+
+
+  useEffect(()=>{
+    fetch(
+      "https://px32id5fdg.execute-api.us-east-1.amazonaws.com/data"
+    )
+    .then((response) => response.json())
+    .then((data) => setMenuData(data))
+  }, [])
```

Comment:
Nice! This approach with `.then()` is great and saves you the hassle of having to define, then invoke, a separate `async` function inside of `useEffect`.

If you had more time, what would you do instead of hard coding the URL on line 12?